README.md

```diff
@@ -209,6 +209,19 @@ Type: `Object`
 
 Sets html attributes and their values on the `menu`. Useful for adding `aria-labelledby` and setting to the value of the `id` attribute on your existing label, to provide context to an assistive technology user.
 
+> [!NOTE]
+>
+> This option will not override the attributes computed by the component
+> - `id` and `role` are dropped to limit risks to accessibility
+> - `className` and `class` are appended to the `class` attribute of the menu so it keeps the classes computed by the component
```

Comment:
I've had a go at re-wording this to include all three `id`, `role` and `onMouseLeave`

Good to explain and link to `menuClasses` too? Don't worry about explaining the `class`/`className` difference since we haven't already done so in the `dropdownArrow` docs

```suggestion
> To maintain assistive technology support, menu attributes `id`, `role` and `onMouseLeave` cannot be overridden using `menuAttributes`. Setting `className` will append to the component default and [`menuClasses`](#menuClasses) values.
```